resources/metadata/detect.yaml

```diff
@@ -303,6 +303,27 @@ spec:
           - PARAMETERS
           - STAGES
           - STEPS
+  outputs:
+    resources:
+      - name: influx
+        type: influx
+        params:
+          - name: step_data
```

Comment:
We should rename `step_data` to `detect_data` to be in line with other steps.

Reply:
this has to be `step_data` ... :-)
It is purely about the step execution flag. This is named the same in other steps.

Reply:
Thanks for confirming Oliver!

Reply:
🙈Sorry mixed up the two